pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "webservices_dispatch_action"
+version = "0.1"
+dynamic = ["dependencies"]
+
+[tool.setuptools]
+packages = ["webservices_dispatch_action"]
+
+[project.scripts]
+run-webservices-dispatch-action = "webservices_dispatch_action.__main__:main"
+run-webservices-dispatch-action-version-updater = "webservices_dispatch_action.version_updater:main"
+
+[tool.flake8]
```

Comment:
This won't work IIRC. See https://pypi.org/project/Flake8-pyproject/

Reply:
OK to move to ruff?

Reply:
Yes OFC! We should add pre-commit as well.